### PR TITLE
Add cli documentation

### PR DESCRIPTION
**Pull request by eheimbuch**
## Proposed changes

Add cli documentation

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

**Contributor**:
- [x] PR is well described and the description can be used as a commit message on squash
- [ ] Related issues linked to PR if existing and labels set
- [ ] New code is covered with unit tests
- [ ] New/updated ui components are tested inside the storybook (module ui-components only) 
- [ ] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog`
- [ ] Feature has been tested with different permissions
- [x] Documentation updated (only necessary for new features or changed behaviour)

**Reviewer**:
- [ ] The clean code principles are respected ([CleanCode](https://clean-code-developer.com/virtues/))
- [ ] All new code/logic is implemented on the right spot / "Should this be done here?"
- [ ] UI changes fits into the layout
- [ ] The UI views / components are responsive (mobile views)
- [ ] Correct translations are available

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: docs/de/user/cli/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: CLI Client
+partiallyActive: true
+---
+
+SCM-Manager bietet einen CLI Client an, um direkt in der Konsole auf z. B. Repositories, Benutzer und Gruppe des SCM-Servers zuzugreifen. 
+
+# Installation / Einrichtung
+Der CLI Client steht für unterschiedliche Betriebssysteme und Architektur auf der offiziellen Webseite zum [Download](https://scm-manager.org/cli/) bereit.
```
Comment (eheimbuch) [inline docs/de/user/cli/index.md:9]:
```suggestion
Der CLI Client steht für unterschiedliche Betriebssysteme und Architekturen auf der offiziellen Webseite zum [Download](https://scm-manager.org/cli/) bereit.
```

File: docs/de/user/cli/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Wählen Sie die passende Installation für sich aus und folgen Sie der Installationsanleitung.
+
+## Anmelden
+Um den CLI Client zu nutzen, muss dieser vorher mit dem SCM-Server verbunden werden. 
+Dazu führen Sie den Befehl `scm login 'https://{server-url:port}/scm'` aus. Ersetzen Sie dabei Ihre Server URL und den Port (falls vorhanden).
+
+### API Schlüssel
+Beim Login wird ein API Schlüssel auf dem Server erzeugt, der für alle weiteren Zugriffe verwendet wird. 
+Dieser API Schlüssel verfügt über alle Rechte, die der angemeldete Benutzer auf dem Server hat. 
+Soll der CLI Client nicht mehr mit dem Server verwendet werden dürfen, reicht es diesen serverseitig zu entfernen.
```
Comment (eheimbuch) [inline docs/de/user/cli/index.md:19]:
```suggestion
Soll der CLI Client nicht mehr mit dem Server verwendet werden dürfen, reicht es diesen Schlüssel serverseitig zu entfernen.
```

File: docs/en/user/cli/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: CLI Client
+partiallyActive: true
+---
+
+SCM Manager provides a CLI client to directly access e.g. repositories, users and group of the SCM server inside your terminal.
+
+# Installation / Setup
+The CLI Client is available for different operating systems and architecture on the official website for [download](https://scm-manager.org/cli/).
```
Comment (eheimbuch) [inline docs/en/user/cli/index.md:9]:
```suggestion
The CLI Client is available for different operating systems and architectures on the official website for [download](https://scm-manager.org/cli/).
```

File: docs/de/user/cli/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Anmelden
+Um den CLI Client zu nutzen, muss dieser vorher mit dem SCM-Server verbunden werden. 
+Dazu führen Sie den Befehl `scm login 'https://{server-url:port}/scm'` aus. Ersetzen Sie dabei Ihre Server URL und den Port (falls vorhanden).
+
+### API Schlüssel
+Beim Login wird ein API Schlüssel auf dem Server erzeugt, der für alle weiteren Zugriffe verwendet wird. 
+Dieser API Schlüssel verfügt über alle Rechte, die der angemeldete Benutzer auf dem Server hat. 
+Soll der CLI Client nicht mehr mit dem Server verwendet werden dürfen, reicht es diesen Schlüssel serverseitig zu entfernen.
+
```
Comment (sdorra) [inline docs/de/user/cli/index.md:20]: We should mention that the cli client will not work if api keys are disabled

File: docs/de/user/cli/index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Mit `scm --help` können die existierenden Befehle auf der obersten Ebene angezeigt werden.
+Ansonsten dokumentiert sich der CLI Client weitgehend selbst. Dazu kann an jeder Stelle die Option `--help` angehangen werden.
+
+## Struktur
+Viele Befehle sind geschachtelt, wobei die oberste Ebene die Resource abbildet, z. B. `repo`, `user` oder `group`.
+Darunter folgen dann die Aktionen wie `create`, `get` oder `list`.
+Neben der Befehlsstruktur unterscheidet der CLI Client noch zwischen `Parametern` als Pflichtargumente und `Optionen` als optionale Argumente.
+
+# Sprache
+Der CLI Client steht in mehreren Sprachen zu Verfügung. Dabei entscheidet die Spracheinstellung des unterliegenden Betriebssystems welche Sprache für die Ausgaben verwendet wird.
```
Comment (sdorra) [inline docs/de/user/cli/index.md:35]: At the moment we support en and de

File: docs/de/user/cli/index.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+# Nutzung
+Die Befehle des CLI Clients werden immer vom Server definiert. Einzig `login` und `logout` sind im CLI Client immer bekannt.
+Mit `scm --help` können die existierenden Befehle auf der obersten Ebene angezeigt werden.
+Ansonsten dokumentiert sich der CLI Client weitgehend selbst. Dazu kann an jeder Stelle die Option `--help` angehangen werden.
+
+## Struktur
+Viele Befehle sind geschachtelt, wobei die oberste Ebene die Resource abbildet, z. B. `repo`, `user` oder `group`.
+Darunter folgen dann die Aktionen wie `create`, `get` oder `list`.
+Neben der Befehlsstruktur unterscheidet der CLI Client noch zwischen `Parametern` als Pflichtargumente und `Optionen` als optionale Argumente.
+
```
Comment (sdorra) [inline docs/de/user/cli/index.md:33]: I think this a good place for a whole section about auto completet

**Review by sdorra — CHANGES_REQUESTED** [review]

**Pull request merged**